odoo/addons/base/models/ir_attachment.py

```diff
@@ -381,6 +381,16 @@ def check(self, mode, values=None):
             if not (self.env.user._is_admin() or self.env.user.has_group('base.group_user')):
                 raise AccessError(_("Sorry, you are not allowed to access this document."))
 
+    @api.model
+    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
+        """
+            Override read_group to add res_field=False in domain if not present.
+        """
+        if not any(item[0] in ('id', 'res_field') for item in domain):
+            domain.insert(0, ('res_field', '=', False))
+        return super(IrAttachment, self).read_group(domain, fields, groupby, offset=offset, limit=limit,
+                                                             orderby=orderby, lazy=lazy)
```

Comment:
Incorrect indentation (it should be aligned with `domain`). I always prefer to break line before the first argument for indenting only one level the arguments, but that's a matter of preference).